integreat_cms/cms/models/pois/poi.py

```diff
@@ -93,6 +94,19 @@ class POI(AbstractContentModel):
         default=get_default_opening_hours,
         verbose_name=_("opening hours"),
     )
+    organization = models.ForeignKey(
+        Organization,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="organization",
```

Comment:
```suggestion
        related_name="pois",
```
The related name is the other way round - it is the names used to fetch all `pois` of an organization object.